### PR TITLE
new example added #127

### PR DESCRIPTION
**Pull request by tosch4**
New example (e5) was added which shows how to modify parameters of simulation model (built in Modelica) with the DymolaAPI.

### CODE REVIEW

**Comment by tosch4** [comment]
Dear all,
updated example e2 and e5 is uploaded into branch #127. Please review.

File: examples/e5_modifier_example.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+"""
+Goals of this part of the examples:
+1. Learn how to use the `DymolaAPI`
+2. Learn how to dynamically modify parameters in the model
+"""
+# Start by importing all relevant packages
+import pathlib
+# Imports from ebcpy
+from ebcpy import DymolaAPI, TimeSeriesData, FMU_API
```
Comment (FWuellhorst) [inline examples/e5_modifier_example.py:9]: FMU_API is not required.

File: examples/e5_modifier_example.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        model_name=ext_model_name,
+        working_directory=working_directory,
+        n_cpu=n_cpu,
+        packages=[aixlib_mo, ibpsa_mo, besmod_mo],
+        show_window=True,
+        n_restart=-1,
+        equidistant_output=False,
+        # Only necessary if you need a specific dymola version
+        dymola_path=r"C:\Program Files\Dymola 2023x",
+        #dymola_version=None
```
Comment (FWuellhorst) [inline examples/e5_modifier_example.py:59]: We can remove these special kwargs

File: examples/e5_modifier_example.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    simulation_setup = {"start_time": 0,
+                        "stop_time": 3600,
+                        "output_interval": 100}
+    dym_api.set_sim_setup(sim_setup=simulation_setup)
+
+    # ######################### Simulation options ##########################
+    # Look at the doc of simulate() in the website
+    # Besides parameters (explained in fmu_example), return_option is important
+
+    # Or change the savepath by using two keyword arguments.
```
Comment (FWuellhorst) [inline examples/e5_modifier_example.py:77]: Change the description, does not fit the current example

File: examples/e5_modifier_example.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    result_sp_2 = dym_api.simulate(
+        return_option="time_series"
+    )
+    print(result_sp_2)
+
```
Comment (FWuellhorst) [inline examples/e5_modifier_example.py:82]: Can you add the following code as well, showing that simulating multiple model_names works, as well? It should work, if not, please give me a note

```

    # You can also simulate a list of different `model_names` (or modified versions of the same model) 
    # by passing a list to the `simulate` function in `DymolaAPI`:
    model_names_to_simulate = [
        "BESMod.Examples.GasBoilerBuildingOnly(redeclare BESMod.Systems.Control.DHWSuperheating control(dTDHW=10))",
        "BESMod.Examples.GasBoilerBuildingOnly(redeclare BESMod.Systems.Control.DHWSuperheating control(dTDHW=5))",
        "BESMod.Examples.DesignOptimization.BES",
    ]
    results = dym_api.simulate(
        return_option="time_series",
        model_names=model_names
    )
    print(results)
```

**Review by FWuellhorst — COMMENTED** [review]
Thanks for the nice example! I suggested minor changes, please have a look

File: examples/e5_modifier_example.py (hunk 4 above)
Comment (tosch4) [reply examples/e5_modifier_example.py:76]: Thanks for your review @FWuellhorst ! Example script is updated und pushed to the issue branch #127

**Review by FWuellhorst — APPROVED** [review]
Nice, thanks for the new example and the work!

**Pull request merged**